internal/oci/runtime_pod.go

```diff
@@ -249,7 +253,7 @@ func (r *runtimePod) AttachContainer(ctx context.Context, c *Container, inputStr
 	)
 
 	if inputStream != nil {
-		stdin = &conmonClient.In{Reader: inputStream}
+		stdin = &conmonClient.In{ReadCloser: io.NopCloser(inputStream)}
```

Comment:
shouldn't Close do something here?

Reply:
Yes we could create our own type and close when we want to. We already close stdin on the client side and I'm not aware of any use case where we would close it in CRI-O. r`